.circleci/config.yml

```diff
@@ -112,7 +112,7 @@ jobs:
 
       - run:
           name: Verify screenshots and generate diffs and new screenshots
-          command: node dockers/screenshotter/screenshotter.js --selenium-ip localhost -b $CIRCLE_JOB --verify --diff --new --coverage
+          command: yarn node dockers/screenshotter/screenshotter.js --selenium-ip localhost -b $CIRCLE_JOB --verify --diff --new --coverage
```

Comment:
https://yarnpkg.com/advanced/migration#call-your-scripts-through-yarn-node-rather-than-node:
> **Call your scripts through `yarn node` rather than `node`**
> We now need to inject some variables into the environment for Node to be able to locate your dependencies. In order to make this possible, we ask you to use `yarn node` which transparently does the heavy lifting.